index.bs

```diff
@@ -340,7 +340,7 @@ An attribution trigger is a [=struct=] with the following items:
 : <dfn>event-level trigger configurations</dfn>
 :: A [=set=] of [=event-level trigger configuration=].
 : <dfn>aggregatable trigger data</dfn>
-:: A [=list=] of [=attribution aggregatable trigger data=].
+:: A [=set=] of [=aggregatable trigger data=].
```

Comment:
The [implementation](https://source.chromium.org/chromium/chromium/src/+/main:content/browser/attribution_reporting/attribution_aggregatable_trigger.h;l=101;drc=3824a643a6687fea3e4f542a30dbc88fadb063b4) doesn't currently represent this as a set. Is that a bug?

Reply:
Thanks Andrew. Yeah, actually probably just keep as list... BTW, the same for event trigger configurations right?

Reply:
Sure, but let's do that in a separate PR.

Reply:
It's originally a list so I just reverted the change to keep as is.